packages/admin/src/pages/super-admin-dashboard/user/[id]/schemes/[schemeId]/change-owner.test.tsx

```diff
@@ -184,6 +184,7 @@ describe('Super admin - Change owner page', () => {
       expect(mockedCheckNewAdminEmailIsValid).toHaveBeenCalledWith(
         'testSessionId',
         'jwt',
+        'test@gmail.com',
         'test@gmail.com'
```

Comment:
Would make these diff emails so its actually making sure its passing in the correct email